lib/full-site-editing/edit-site-page.php

```diff
@@ -101,7 +101,10 @@ function gutenberg_edit_site_init( $hook ) {
 			'__experimentalBlockPatternCategories' => WP_Block_Pattern_Categories_Registry::get_instance()->get_all_registered(),
 		)
 	);
-	$settings = gutenberg_experimental_global_styles_settings( $settings );
+
+	$site_editor_context = new WP_Block_Editor_Context();
```

Comment:
I'm not quite sure why the edit site page doesn't integrate this filter like other screens. If we decide to change it, we should refactor the code to use `gutenberg_get_block_editor_settings` to handle it all behind the scenes rather that duplicating a similar logic.

Reply:
Makes sense to me. I'll make the changes. 👍

Reply:
Updated in 72b175c